docs/windows.md

```diff
@@ -171,18 +171,52 @@ curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/lates
 .\PrepareNode.ps1 -KubernetesVersion v1.18.0
 ```
 
-4. Prepare network adapter for kube-proxy
+4. Prepare node environment needed by antrea-agent
 
-kube-proxy needs a network adapter to configure Kubernetes Service IPs and uses
-the adapter for proxying connections to Service. Use following script to create the network
-adapter.
+Run following commands to prepare node environment needed by antrea-agent:
 ```
+mkdir c:\k\antrea
+cd c:\k\antrea
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Clean-AntreaNetwork.ps1
 curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Prepare-ServiceInterface.ps1
-.\Prepare-ServiceInterface.ps1
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Prepare-AntreaAgent.ps1
+.\Prepare-AntreaAgent.ps1
 ```
+The script `Prepare-AntreaAgent.ps1` completes following tasks:
+- Prepare network adapter for kube-proxy.
+
+    kube-proxy needs a network adapter to configure Kubernetes Service IPs and
+    uses the adapter for proxying connections to Service. Use following script
+    to create the network adapter. The adapter will be deleted automatically by
+    Windows after Windows Node reboots.
+
+- Remove stale network resources created by antrea-agent.
+
+    After Windows Node reboots, there will be stale network resources which
+    need to be cleaned before starting antrea-agent.
+
+> Note: The script must be executed evey time the node is restarted.
+
+As you know from the task details from above, the script must be executed every
+time you restart the Node to prepare the environment for antrea-agent.
 
-> Note: The interface will be deleted automatically by Windows after Windows
-> Node reboots. So the script needs to be executed after rebooting the Node.
+You could make the script be executed automatically after Windows startup by
+some methods. Here're two examples for your reference:
+
+- Example1: Update kubelet service.
+
+Insert following line in kubelet service script `c:\k\StartKubelet.ps1` to invoke
+`Prepare-AntreaAgent.ps1` when starting kubelet service:
+```
```

Comment:
leave an empty line before code blocks for the website

also adds the appropriate language tag, e.g. `powershell`